docs/self-managed/zeebe-deployment/kubernetes/index.md

```diff
@@ -29,3 +35,37 @@ You will need to configure your Kubernetes cluster and modify this to suit the a
 Zeebe gateway is deployed as a stateless service.
 
 We support [Kubernetes startup and liveness probes](../operations/health.md#gateway) for Zeebe gateway.
+
+### Helm
+
+There are several alternatives to deploy applications to a Kubernetes cluster, but the following sections use Helm charts 
+to deploy a set of components into your cluster.
+
+Helm allows you to choose exactly what chart (set of components) you want to install and how these components need to be configured.
+
+These Helm charts are continuously being improved and released to the [Camunda Cloud Helm Chart Repository](https://github.com/camunda-community-hub/camunda-cloud-helm).
+
+You are free to choose your Kubernetes provider as our Helm charts are not cloud provider-specific.
+
+We encourage [reporting issues](https://github.com/camunda-community-hub/camunda-cloud-helm/issues) if you find them.
+
+## Prerequisites
+
+To use Kubernetes, you must have the following tools installed in your local environment:
+
+- `kubectl`: Kubernetes Control CLI tool, installed and connected to your cluster
+- `helm`: Kubernetes Helm CLI tool
+
+You also need a Kubernetes cluster. You have several options:
+
+- Local for development, you can use [Kubernetes KIND](https://github.com/kubernetes-sigs/kind), Minikube, and MicroK8s.
+- Remote: Google GKE, Azure AKS, Amazon EKS, etc.
```

Comment:
:wrench: Be aware that we only guarantee or test gke at the moment

Reply:
Added this to a note below. 👍